### PR TITLE
Pull request/abba71cc

### PR DESCRIPTION
**Pull request by yakir12**
Yea, I forgot to fix the docstring.. 
I figured that `dest2src = nothing` is simpler and makes more sense so no `like_identity` is needed. The default behavior of `bind!` for `twoway` results in `dest2src = identity` anyhow so in practice that's what happens except when the user wants to modify the binded map in a specific way.

### CODE REVIEW

File: test/basics.jl [hunk 1]
```diff
@@ -298,6 +298,90 @@ facts("Basic checks") do
...
+        @fact value(dst) --> 5
+        push!(dst,1)
+        step()
+        @fact value(src) --> -1
+
+        # This should work but I think line 444 in operators.jl shouldn't exit if the twoway unbind! was applied after a oneway unbind!.
+        # unbind!(dst,src) # test other way as well
+        # push!(dst,2)
+        # step()
+        # @fact value(src) --> -1
```
Comment (JobJob) [inline test/basics.jl]: Should this work now?

File: src/operators.jl [hunk 2]
```diff
@@ -353,7 +370,7 @@ function bind!(dest::Signal, src::Signal, twoway=true; initial=true)
             # `true` below is for dont_remove_dead nodes - messes with active_nodes
             # TODO - check that - not sure it actually does, this may be a relic
             # of an earlier implementation which used the node's id's
-            run_push(dest, src.value, onerror_rethrow, true)
+            run_push(dest, src2dest(src.value), onerror_rethrow, true) # here is the only place where we implement the modifying function!
```
Comment (JobJob) [inline src/operators.jl]: nice 👍

File: test/basics.jl [hunk 3]
```diff
@@ -298,6 +298,90 @@ facts("Basic checks") do
...
+        push!(src, 2)
+        step()
+        @fact value(dst) --> 4
+        push!(dst, 5)
+        step()
+        @fact value(src) --> 2
+        unbind!(dst,src)
+        push!(src,1)
+        step()
+        @fact value(dst) --> 5
```
Comment (JobJob) [inline test/basics.jl]: 👍

**Comment by JobJob** [comment]
This is nice in the end - good work!

**Comment by yakir12** [comment]
Sweet! Very happy to help. All my best work comes from copy-pasting other's work...

File: test/basics.jl (hunk 1 above)
Comment (yakir12) [reply test/basics.jl]: I don't think so. [This line](https://github.com/JuliaGizmos/Reactive.jl/blob/5d1a8da7c6053330f2db31df6febdea028dfbe9c/src/operators.jl#L403), I think, only checks src=>dest but not dest=>src? so it just exists even if the other way around existed...

**Comment by JobJob** [comment]
😄

**Pull request merged**